pkgs/applications/video/gnome-mpv/default.nix

```diff
@@ -1,25 +1,30 @@
-{ stdenv, fetchurl
-, intltool, pkgconfig, wrapGAppsHook
+{ stdenv, fetchFromGitHub, meson, ninja
+, gettext, pkgconfig, desktop_file_utils, wrapGAppsHook
 , appstream-glib, epoxy, glib, gtk3, mpv
 }:
 
 stdenv.mkDerivation rec {
   name = "gnome-mpv-${version}";
-  version = "0.9";
+  version = "0.13";
 
-  src = fetchurl {
-    sha256 = "06pgxl6f3kkgxv8nlmyl7gy3pg55sqf8vgr8m6426mlpm4p3qdn0";
-    url = "https://github.com/gnome-mpv/gnome-mpv/releases/download/v${version}/${name}.tar.xz";
+  src = fetchFromGitHub {
+    owner = "gnome-mpv";
+    repo = "gnome-mpv";
+    rev = "v${version}";
+    sha256 = "0ismh56k0alzplhpimzb2wlhj0bgqiw4hsqrz917vdgly1959ghg";
   };
 
-  nativeBuildInputs = [ intltool pkgconfig wrapGAppsHook ];
-  buildInputs = [ appstream-glib epoxy glib.dev gtk3 mpv ];
-
-  NIX_CFLAGS_COMPILE = "-I${glib.dev}/include/gio-unix-2.0";
+  nativeBuildInputs = [ meson ninja appstream-glib gettext pkgconfig desktop_file_utils wrapGAppsHook ];
```

Comment:
Could you remind me, when should `ninja` be specified along with `meson`, and when `meson` is enough without `ninja`?

Reply:
We want to always use `ninja` with `meson` – meson just generates build instructions for ninja. Though, in this case it might work without ninja because the package still supports autoconf.